WcaOnRails/app/models/locale.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+class Locale < SimpleDelegator
+  include ActionView::Helpers::TextHelper
+  SOME_CHARS = '[\s\S]*?'
```

Comment:
@viroulep What do you mean by the optional match? It's just non-greedy, and it's not for performance in this case, it won't work the other way.
Par example
```yaml
en:
  common:
    competitions: "Competitions"
  competitions:
    venue: "Venue"
```
If our regexps wouldn't use non-greedy matchers, then trying to match the key `en.common.competitions`, would match the `competitions:` tree.